pupil-spa/src/app/footer/footer.component.html

```diff
@@ -6,7 +6,6 @@ <h2 class="visuallyhidden">Support links</h2>
         <ul>
           <li><a routerLink="/contact">Contact</a></li>
           <li><a target="_blank" routerLink="/accessibility-statement">Accessibility Statement</a></li>
-          <li><a target="_blank" routerLink="/privacy">Privacy notice</a></li>
```

Comment:
The old privacy page will still be visible should anyone just type (or click a link) with '/privacy' on the path.  As the copy would be out of date it would be preferable to redirect to the new location at the component level.